src/test/java/seedu/address/logic/commands/AddCommandTest.java

```diff
@@ -133,6 +139,11 @@ public void deleteClient(Client target) {
             throw new AssertionError("This method should not be called.");
         }
 
+        @Override
+        public void deleteProcedure(Procedure procedure) {
```

Comment:
Again, how about separating it into a new class and write tests inside it?

Reply:
Hi JB, Oh yes I agree. This method exists as a separate class, but it has to be implemented because in this ModelStub because a ModelStub implements from a Model, which has the following functions as well.